<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Evan Teran</author>
 <class>ProcessPropertiesPlugin::DialogStrings</class>
 <widget class="QDialog" name="ProcessPropertiesPlugin::DialogStrings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>String Search</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblRegions">
     <property name="text">
      <string>Regions To Search:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtSearch">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Filter</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="font">
      <font>
       <family>Monospace</family>
      </font>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="search_unicode">
     <property name="text">
      <string>Include UTF-16 Results</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProcessPropertiesPlugin::DialogStrings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>644</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>671</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProcessPropertiesPlugin::DialogStrings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>532</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>516</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
